step.yml

```diff
@@ -8,21 +8,24 @@ description: |-
 
   ### Configuring the Step
 
-  1. Set the value for the **Deploy directory or file path** required input. It can be a path to a directory or a file. If you provide a directory, everything in that directory gets uploaded recursively. 
-  If you provide only a file, then only that file gets uploaded. The default value is the `$BITRISE_DEPLOY_DIR` Env Var which is exposed by the Bitrise CLI.
+  1. Set the value for the **Deploy directory or file path** required input. The default value is the `$BITRISE_DEPLOY_DIR` Env Var which is exposed by the Bitrise CLI.
+  It can be a path to a directory or a file. 
+  If you provide a directory, everything in that directory, excluding sub-directories, gets uploaded recursively. 
```

Comment:
Everything in the directory gets uploaded and nothing deeper. so recursively is not needed.